packages/rocketchat-i18n/i18n/es.i18n.json

```diff
@@ -320,8 +321,14 @@
   "API_GitHub_Enterprise_URL": "URL del servidor",
   "API_GitHub_Enterprise_URL_Description": "Ejemplo: http://domain.com (sin incluir la diagonal final)",
   "API_Gitlab_URL": "URL de GitLab",
-  "API_Personal_Access_Tokens_Regenerate_It": "Regenerar token",
-  "API_Shield_Types": "Tipos de Escudo",
+  "API_Personal_Access_Token_Name": "Nombre de Token de Acceso Personal ",
+  "API_Personal_Access_Tokens_To_REST_API": "Token de acceso personal a API REST",
+  "API_Personal_Access_Tokens_Remove_Modal": "¿Estás seguro que quieres eliminar este token de acceso personal?",
+  "API_Personal_Access_Token_Generated": "Token de Acceso Personal generado con éxito",
+  "API_Personal_Access_Token_Generated_Text_Token_s_UserId_s": "Por favor guarda cuidadosamente tu token porque no podrás visualizarlo después. <br/>Token: <strong>__token__</strong><br/>Tu ID de usuario: <strong>__userId__</strong>",
+  "API_Personal_Access_Tokens_Regenerate_Modal": "Si perdiste tu token, puedes volver a generarlo, pero recuerda que las aplicaciones que lo usen tendrán que actualizarlo",  
+  "API_Personal_Access_Tokens_Regenerate_It": "Regenerar token",  
```

Comment:
Added whitespace to the end of the line makes git think that this line was edited, when it wasn't. These kind of files are not linted automatically, but regardless, try and remove unnecessary changes.

Reply:
Thank you for your feedback @gabriellsh!

These changes are new lines added on translation file. This is what I see by comparing the versions:

![image](https://user-images.githubusercontent.com/3056856/71727519-cec1a700-2e3a-11ea-886f-11f5c49cfde1.png)